e2e/integrations/integration.steps.ts

```diff
@@ -100,107 +101,118 @@ class IntegrationSteps {
   }
 
   @then(/^she fill configure page for "([^"]*)" step with "([^"]*)" parameter$/)
-  public fillStepConfiguration(stepType: string, paremeter: string): P<any> {
+  public fillStepConfiguration(stepType: string, parameter: string): P<any> {
     const stepFactory = new StepFactory();
-    const page = stepFactory.getStep(stepType, paremeter);
+    const page = stepFactory.getStep(stepType, parameter);
     return page.fillConfiguration();
   }
 
   @then(/^she adds "([^"]*)" random steps and then check the structure$/)
-  public addRandomStepsAndCheckRest(numberOfSteps: number): void {
+  public async addRandomStepsAndCheckRest(numberOfSteps: number): P<any> {
+    log.info(`Adding random steps`);
 
-    this.getStepsArray().then((array) => {
-      this.world.app.clickButton('Add a Step');
+    const array = await this.getStepsArray();
+
+    this.world.app.clickButton('Add a Step');
+
+    const links = this.world.app.getLinks('Add a step');
+    const count = await links.count();
+
```

Comment:
Are there links and buttosn that has different capitalization of `Step`? If that's the case, please create new issue in UI repo.